dev-haskell/hpack/hpack-0.35.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.7.1.2
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A modern format for Haskell packages"
+HOMEPAGE="https://github.com/sol/hpack#readme"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-haskell/aeson-1.4.3.0:=[profile?]
+	dev-haskell/bifunctors:=[profile?]
+	>=dev-haskell/cabal-3.0.0.0:=[profile?] <dev-haskell/cabal-3.7:=[profile?]
+	dev-haskell/cryptonite:=[profile?]
+	>=dev-haskell/glob-0.9.0:=[profile?]
+	dev-haskell/http-client:=[profile?]
+	dev-haskell/http-client-tls:=[profile?]
+	dev-haskell/http-types:=[profile?]
+	>=dev-haskell/infer-license-0.2.0:=[profile?] <dev-haskell/infer-license-0.3:=[profile?]
+	dev-haskell/scientific:=[profile?]
+	dev-haskell/unordered-containers:=[profile?]
```

Comment:
Its missing the text dependency:
````
	>=dev-haskell/text-0.11:=[profile?]
````

Reply:
text is marked as an un-upgradeable bundled library now, so hackport doesn't write it.

Reply:
@solpeth so I shouldn't add it, all fine?

Reply:
yep, leave it out and that should be fine.

Reply:
Great, thanks!